packages/firebase_auth/firebase_auth_web/lib/src/utils/web_utils.dart

```diff
@@ -281,13 +285,6 @@ auth_interop.OAuthCredential? convertPlatformCredential(
     );
   }
 
-  if (credential is OAuthCredential) {
```

Comment:
This should be down here and the other one removed shouldn't it, since `TwitterAuthCredential` is an instance of `OAuthCredential` it will get caught in the one above `if (credential is TwitterAuthCredential)` won't it?